### DISCUSSION TITLE
Where does the "update-available.ini" file come from?

### DISCUSSION THREAD

**Discussion by PetersOtto** [datenstrom/community#942] [Ask a question] [answered]
Hello!

I have a small question about the update extension.

The update extension compares the entries of `update-available.ini` and `update-current.ini`. If the entries differ, an update starts. Is this correct?

If I want to update my own extensions via the browser, how do I get the information, that there is a new version into the `update-available.ini`?

**Comment by markseuffert** [comment, marked as answer]
The file `update-available.ini` is downloaded from the repository `datenstrom/yellow`. The update mechanism compares version numbers and downloads extensions if a higher version number is available. To make your extension part of the update mechanism it needs to be published. Publishing means from a purely technical perspective that the version number is increased and the necessary files are updated in the repository `datenstrom/yellow`. [Learn more about making and publishing extensions](https://github.com/annaesvensson/yellow-publish).

Hope this helps and see also datenstrom/community#892.

**Reply by PetersOtto** [reply]
Ok, thanks for the information!

**Reply by markseuffert** [reply]
I edited the file name to make this answer easier to find. Hope that's okay with you.

**Reply by PetersOtto** [reply]
Of course!
